sdks/java/core/src/main/java/org/apache/beam/sdk/state/MapState.java

```diff
@@ -56,12 +56,6 @@
    * <p>Changes will not be reflected in the results returned by previous calls to {@link
    * ReadableState#read} on the results any of the reading methods ({@link #get}, {@link #keys},
    * {@link #values}, and {@link #entries}).
-   *
-   * <p>Since the condition is not evaluated until {@link ReadableState#read} is called, a call to
```

Comment:
We'll need a follow-up change fixing the other implementations of MapState to perform the correct order of actions for `putIfAbsent` and `computeIfAbsent`.

Reply:
https://issues.apache.org/jira/browse/BEAM-12922

I'll take a look after this